staging/src/k8s.io/component-helpers/node/util/helpers.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
`k8s.io/component-helpers/node` directory respects the sig name. Though, I'd like to avoid using `util` keyword in the path. Can you use `k8s.io/component-helpers/node/failuredomain` or similar that will name a group of semantically close helpers together?

Reply:
+1 to not using util as the pkg name, `topology` sounds reasonable to me.

Reply:
Sure, updated to `topology`

Reply:
Forgot to change `k8s.io/component-helpers/node/util`

Reply:
Ah yeah, forgot to actually run `make update`. Done

Reply:
`vendor/modules.txt` still has `k8s.io/component-helpers/node/util` listed